mycroft/util/parse.py

```diff
@@ -164,6 +169,571 @@ def extractnumber_en(text):
     return val
 
 
+'''authors: spenrod and connorpenrod
+this function takes a sentence and extracts date and time information from it
+returns a 2-element array [date, leftover_string]
+the 'date' is a python datetime object
+the 'leftover_string' contains all keywords not related to the date or time'''
```

Comment:
I'd modify the 'leftover_string' description a smidgeon for clairity.  More like:
   leftover_string (str):  the original phrase minus all words related to the datetime

In the longer description of the function, include an example.  Something simple like:
     extract_date_time_en("turn on the light at 3pm") will returns "turn on the light" as the leftover string